org.kde.kget.json

```diff
@@ -0,0 +1,135 @@
+{
+  "id" : "org.kde.kget",
+  "runtime" : "org.kde.Platform",
+  "runtime-version" : "5.15-21.08",
+  "sdk" : "org.kde.Sdk",
+  "command" : "kget",
+  "rename-icon" : "kget",
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--filesystem=host",
```

Comment:
I am of the opinion this should be limited to `xdg-download`

Reply:
i will need little bit explanation too

Reply:
The idea is that KGet does not need to access your entire home directory. It just needs a place to download files and xdg-download is just that (see the Freedesktop spec).

Reply:
understood. thanks